community/common/src/main/java/org/neo4j/function/Predicates.java

```diff
@@ -155,6 +155,23 @@ public static void await( Supplier<Boolean> condition, long timeout, TimeUnit un
         await( condition, timeout, unit, defaultPollInterval, TimeUnit.MILLISECONDS );
     }
 
+    public static void awaitEx( ThrowingSupplier<Boolean, Exception> condition, long timeout, TimeUnit unit )
```

Comment:
This method should be called `await` because it is an overload of the following one.